src/test/java/com/easypost/AddressTest.java

```diff
@@ -70,11 +78,14 @@ public void testCreateVerifyStrict() throws EasyPostException {
      */
     @Test
     public void testRetrieve() throws EasyPostException{
-        Address retrievedAddress = Address.retrieve(globalAddress.getId());
+        _vcr.setUpTest("retrieve");
+
+        Address address = createBasicAddress();
+        Address retrievedAddress = Address.retrieve(address.getId());
+
+        assertInstanceOf(Address.class, retrievedAddress);
+        assertEquals(address.getId(), retrievedAddress.getId());
 
-        assertTrue(retrievedAddress instanceof Address);
-        assertTrue(retrievedAddress.getId().startsWith("adr_"));
-        assertThat(globalAddress).usingRecursiveComparison().isEqualTo(retrievedAddress);
```

Comment:
Was this removed because we can no longer compare objects due to cassette scrubbing?

Reply:
Unit test was made to match C#